### DISCUSSION TITLE
Domain falsely blocked (nyxiaai.com)

### DISCUSSION THREAD

**Discussion by sergiu-nichitean** [phantom/docs#231] [Help & Support] [answered]
### Summary

Hi, we are Nyxia AI and currently building our AI app where you can generate songs with AI and mint it as an NFT.

Our site gets falsely flagged, both the main domain and related sub-domains like title.nyxiaai.com (that's part of the feature from the mint site builder).

We have already opened an issue on the blocklist repository, but we got no answer in 9 days: https://github.com/phantom/blocklist/issues/1122

Also, we checked the blocklist and we are not present there, so not sure who is blocking us: https://github.com/phantom/blocklist/blob/master/blocklist.yaml

We have a TG group where we manage app testing and you can see the issue reported there: https://t.me/+-HyXQo_6H-8zOThi

![Untitled](https://github.com/phantom/docs/assets/3617390/baef9415-94dc-4710-bd65-411a56a0871b)

### Example

https://builder.nyxiaai.com/

### Steps to Reproduce

Go to https://builder.nyxiaai.com/ or any other subdomain while having the Phantom extension enabled and you get a message that the website is blocked.

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
resolved
